server/StudentList.ts

```diff
@@ -40,5 +41,12 @@ class StudentList extends EventEmitter {
         return (student1.name === student2.name && student1.surname === student2.surname && student1.nick === student2.nick)
     }
 
+    includesStudent(student: Student): boolean{
```

Comment:
It is StudentList it only should include Students.
I propose name: `includes`

Reply:
u r ->

Reply:
you are right xD